procurement/filters.py

```diff
@@ -122,4 +122,3 @@ def filter_notification_frequency(self, queryset, name, value):
             for future_day in NOTIFICATION_MONTHS:
                 possible_end_dates.append(day + future_day)
         return queryset.filter(end_date__in=possible_end_dates)
-<<<<<<< HEAD
```

Comment:
would be nice to fix this in the commit it was introduced rather than here.

Reply:
This has now been fixed with a rebase.